pages/static/assets/common/overview.svg

```diff
@@ -0,0 +1,53 @@
+<svg width="296" height="288" viewBox="0 0 296 288" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
This one also

Reply:
This one is not ready, Overview component excluded for now